src/libraries/System.Security.Cryptography.OpenSsl/tests/System.Security.Cryptography.OpenSsl.Tests.csproj

```diff
@@ -1,9 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
-    <!-- ActiveIssue Apple Silicon No usable version of libssl was found
-        https://github.com/dotnet/runtime/issues/49083 -->
```

Comment:
Based on the comment I'm guessing that the CI images either a) don't have OSSL or b) have x86_64 OSSL which can't be loaded by an arm64 CLR.

But if the tests pass in CI, I'm cool with removing the block.

Reply:
If it does not, I'll work it out with infra team. now I know for sure `Brew` _can_ do what we need.